src/services/kratos/auth-phone-no-password.ts

```diff
@@ -0,0 +1,143 @@
+import { getKratosMasterPhonePassword } from "@config"
+
+import {
+  AuthenticationKratosError,
+  IncompatibleSchemaUpgradeError,
+  KratosError,
+  LikelyNoUserWithThisPhoneExistError,
+  LikelyUserAlreadyExistError,
+  UnknownKratosError,
+} from "@services/kratos/errors"
+import {
+  AdminCreateIdentityBody,
+  AdminUpdateIdentityBody,
+  SuccessfulSelfServiceLoginWithoutBrowser,
+  SuccessfulSelfServiceRegistrationWithoutBrowser,
+} from "@ory/client"
+
+import { AxiosResponse } from "node_modules/@ory/client/node_modules/axios/index"
+
+import { kratosAdmin, kratosPublic, toDomainIdentity } from "./private"
+
+// login with phone
+const password = getKratosMasterPhonePassword()
+
+export const AuthWithPhoneNoPassword = (): IAuthWithPhoneNoPassword => {
```

Comment:
Still not sure about this name. Is it perhaps a `PaswordlessAuthenticationService`? Having something like `XxxService` is just to stay consistent with all the rest of our `ExternalService` implementations.